src/nvim/main.c

```diff
@@ -927,9 +926,6 @@ static void parse_command_name(mparm_T *parmp)
   if (parse_char_i(&initstr, 'r'))
```

Comment:
Yes, it's about 10 lines before currently.

On January 21, 2015 12:16:54 AM EST, "Justin M. Keyes" <notifications@github.com> wrote:
>> @@ -927,9 +926,6 @@ static void parse_command_name(mparm_T *parmp)
>>    if (parse_char_i(&initstr, 'r'))
>
>@Pyrohh @fmoralesc I assume line 906 was moved after the rebase, so
>this comment was addressed?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/neovim/neovim/pull/1656/files#r23278514